app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -410,3 +410,16 @@ export default {
   },
 };
 </script>
+
+<style lang="scss" scoped>
+ @import '~bootstrap/dist/css/bootstrap.min.css';
+ @import 'https://unpkg.com/formiojs@4.11.2/dist/formio.builder.min.css';
```

Comment:
Yeah this looks promising. If we can only import the bootstrap on the components scoped to hold the form designer that might be the best. If this works out, ideally we wrap the form io component in a slimmer component so the bootstrap is at the lowest level it needs to be. Can coordinate with Jeremy since he's moving the builder/designer components around as it is.

Reply:
great. yeah, this bit seems to be working. i'll commit some more changes now for review.. because there seems to be a simpler way of overriding the formio styles.